<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectTransfer</class>
 <widget class="QWidget" name="DirectTransfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DirectTransfer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>文件传输与格式化</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="selectAll">
        <property name="text">
         <string>全选/取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="sdWidget" native="true"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="startFormat">
       <property name="text">
        <string>格式化</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="selectSavePath">
       <property name="text">
        <string>选择存储路径</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="startTransfer">
       <property name="minimumSize">
        <size>
         <width>119</width>
         <height>34</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>119</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>开始传输</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
